src/controllers/AbstractController.ts

```diff
@@ -45,11 +45,27 @@ type SidecarRequestHandler =
 	| RequestHandler<IParaIdParam>
 	| RequestHandler;
 
+/*
+ * The required pallets for a controller can be a list of pallets in string, or a definition using AND | OR and the list of pallets
+ * Example:
+ * 		- ['pallet1', 'pallet2']
+ * 		- { 'OR': ['pallet1', 'pallet2'] }
+ * 		- { 'AND': ['pallet1', 'pallet2'] }
```

Comment:
In what scenario is the usage of `AND` expected? Is not providing a list with every pallet enough to cover this case?